### PR TITLE
linters + change imports style

### PR DESCRIPTION
**Pull request by ivan-savciuc**
This PR updates golang imports appearance and updated GitHub workflow to validate that, also includes other checks as well

### CODE REVIEW

File: aiven/provider.go [hunk 1]
```diff
@@ -5,14 +5,15 @@ package aiven
...
-	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"log"
 	"net/url"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/aiven/terraform-provider-aiven/pkg/cache"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
```
Comment (mhoffm-aiven) [inline aiven/provider.go:16]: this whitespace is weird

**Review by mhoffm-aiven — CHANGES_REQUESTED** [review]
There are some files left where the external imports are not grouped properly ( see comment ). All in all loving it though.

File: aiven/provider.go (hunk 1 above)
Comment (ivan-savciuc) [reply aiven/provider.go:16]: I did `goimports  -w .` and as you can see `golangci-clint -E goimports` is also has no complaints. How different should this be from what we have?

Comment (mhoffm-aiven) [reply aiven/provider.go:16]: i think goimports does not care about grouping of the imports; i think we should group all external imports together ( its only 3 files that need this post processing )

**Review by mhoffm-aiven — APPROVED** [review]
looks good to me

**Pull request merged**